data_juicer/ops/mapper/mllm_mapper.py

```diff
@@ -0,0 +1,113 @@
+import torch
+
+from data_juicer.ops.base_op import OPERATORS, Mapper
+from data_juicer.ops.op_fusion import LOADED_IMAGES
+from data_juicer.utils.mm_utils import load_image
+from data_juicer.utils.model_utils import get_model, prepare_model
+
+OP_NAME = 'mllm_mapper'
+torch.set_num_threads(1)
+
+
+@LOADED_IMAGES.register_module(OP_NAME)
+@OPERATORS.register_module(OP_NAME)
+class MllmMapper(Mapper):
```

Comment:
Is this op a bit similar to `image_captioning_mapper `?
It is recommended to add the prefix 'Image'.

Reply:
This mapper is designed to invoke MLLMs to address multimodal tasks involving both images and text. I forgot to update the annotation here.